Cannot make the application

When I run `make install`, I get the following output, and it never installs. I'm not sure how to debug this.

```
make -C src/
make[1]: Entering directory `/hostonly/svn/myuser/dev/todo.c/src'
cc    -c -o main.o main.c
cc    -c -o hbuf.o hbuf.c
cc    -c -o generator.o generator.c
cc    -c -o parser.o parser.c
cc    -c -o todo.o todo.c
cc    -c -o utils.o utils.c
cc main.o hbuf.o generator.o parser.o todo.o utils.o -o todo -I./
make[1]: Leaving directory `/hostonly/svn/myuser/dev/todo.c/src'
test -d /home/myuser/bin
test -d /home/myuser/man/man1
make: *** [dirs] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -16,12 +16,11 @@ clean:
 		$(MAKE) -C src/ clean
 		rm -f todo.1
 
-dirs:
-		test -d $(BINDIR)
-		test -d $(MAN1DIR)
 
-install: compile dirs todo.1
+install: compile todo.1
+		install -d $(BINDIR)
 		install src/todo $(BINDIR)
+		install -d $(MAN1DIR)
 		install todo.1 $(MAN1DIR)
 
 uninstall:

